.github/workflows/build-and-test.yml

```diff
@@ -1390,7 +1396,7 @@ jobs:
     name: "Upload STT packages to npmjs.com"
     runs-on: ubuntu-20.04
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
-    needs: [create-release, repackage-nodejs-allplatforms]
+    needs: [create-release, repackage-nodejs-allplatforms, build-wasm]
```

Comment:
why is it part of it ?

Reply:
Most likely introduced into main with https://github.com/coqui-ai/STT/commit/7f50e0a9b51473bbed57d0263d286cd85f5a5238

Reply:
Fixed